(GH-2219) Add 'module' and 'modulepath' topic guides (#2279)
puppetlabs/bolt: guides/modulepath.txt

TOPIC
    modulepath

DESCRIPTION
    The modulepath is an ordered list of directories that Bolt loads modules
    from. When Bolt runs a command, it automatically loads modules from the
    modulepath.
    
    While Bolt has a default modulepath, you can also configure your own
    modulepath, which can include directories within the project or directories
    elsewhere on your system. Regardless of whether your project uses a default

>     or configured modulepath, Bolt automatically adds  directories to the

Reviewer: ```suggestion
    or configured modulepath, Bolt automatically adds directories to the
```

>     or configured modulepath, Bolt automatically adds a couple directories to

Reviewer: ```suggestion
    or configured modulepath, Bolt automatically adds two directories to
```
It's just the two now right? If it's a small, but undefined number, I would maybe use different phrasing - For example "certain directories" / "specific directories" or forego the descriptor and just use "...adds directories...".

Author: _ish_. It adds `.modules` if you're using the module management workflow, and the project if you're using a named project. I'll be less specific.

    modulepath. This includes modules containing core Bolt content, which is
    added to the beginning of the modulepath, and bundled content, which is
    added to the end of the modulepath.

    Modules loaded from a directory listed earlier in the modulepath take
    precedence over modules with the same name loaded from a directory later in
    the modulepath. Bolt will not warn or error when two modules share a name

>     and instead will shadow modules with a lower precedence.

Reviewer 2: ```suggestion
    and instead will ignore modules with a lower precedence.
```

Reviewer: Ah, good catch!
